frontend/src/components/posts/post_show.jsx

```diff
@@ -106,12 +106,7 @@ class PostShow extends Component {
           <div className="video-header">
             <div className="left-side-header">
               <p className="show-title">{post.title}</p>
-              {/* <p>by <Link
-                className='profile-link'
-                to={`/profile/${post.user._id}`}
-              >{post.user.username}</Link>
-              </p> */}
-              <p>by {post.user.username}</p>
+              <p className="author">by {post.user.username}</p>
```

Comment:
I was thinking of keeping this comment in, since we can immediately uncomment it once the profile pages are ready.